doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -237,6 +237,9 @@ Non-resilient applications are sensitive to platform impairments on Compute like
 
 <p align="center"><b>Table 3-1:</b> Categories of applications, requirements for scheduling pods and Kubernetes features</p>
 
+Kubernetes clusters with above enhancements can run on bare metal servers (directly on Linux without host Operating System) or in virtual machines (VMs, on hypervisor).
```

Comment:
"directly on Linux without host Operating System" - do you mean "with" instead of "without"?

Reply:
> "directly on Linux without host Operating System" - do you mean "with" instead of "without"?

@tomkivlin : Yes. Hope it is more precise as `can run "bare metal" (directly on Linux host Operating System)`. Thanks